packages/protocol/runTests.js

```diff
@@ -35,6 +35,7 @@ const migrationOverrides = {
   },
   reserve: {
     initialBalance: 100000000,
+    otherAddresses: [network.from],
```

Comment:
Assuming this is to test actually transferring gold with the reserve spender? It might be more readable to put an arbitrary, hard-coded address, and include a comment as to what this is for, e.g.

```
// Add an arbitrary "otherReserveAddress" so that reserve spending can be tested.
```